packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -494,10 +593,16 @@ abstract class SchedulerBinding extends BindingBase {
   /// another frame to be scheduled, even if the current frame has not yet
   /// completed.
   ///
+  /// If `force` is set to true, then the frame will be scheduled even if the
+  /// device's screen is turned off or the application is otherwise not visible.
+  /// The framework uses this to force a frame to be rendered at the correct
+  /// size when the phone is rotated, so that a correctly-sized rendering is
+  /// available when the screen is turned back on.
+  ///
   /// To have a stack trace printed to the console any time this function
   /// schedules a frame, set [debugPrintScheduleFrameStacks] to true.
-  void scheduleFrame() {
-    if (_hasScheduledFrame)
+  void scheduleFrame({ bool force: false }) {
```

Comment:
scheduleFrame might be pretty hot.  Consider creating two entry points rather than adding a named parameter.  For example, `scheduleFrame()` that checks the dirty bits and then calls `forceScheduleFrame`, which schedules the frame unconditionally.

Reply:
Done